test/models/solid_cache/entry_test.rb

```diff
@@ -70,9 +70,33 @@ class EntryTest < ActiveSupport::TestCase
       end
     end
 
+    test "batching multi queries" do
+      with_multi_batch_size(2) do
+        Entry.stubs(:const_get).with(:MULTI_BATCH_SIZE).returns("stubbed_value")
+
+        assert_queries_count(2) do
+          Entry.write_multi([ { key: "hello".b, value: "there" }, { key: "foo".b, value: "bar" }, { key: "baz".b, value: "zab" } ])
+        end
+
+        assert_queries_count(2) do
+          assert_equal({ "foo" => "bar", "hello" => "there", "baz" => "zab" }, Entry.read_multi([ "hello".b, "foo".b, "baz".b, "bar".b ]))
+        end
+      end
+    end
+
     private
       def write_entries(count = 20)
         Entry.write_multi(count.times.map { |i| { key: "key#{i}", value: "value#{i}" } })
       end
+
+      def with_multi_batch_size(value)
```

Comment:
How about using `stub_const`? This logic is almost the same as that method.
https://api.rubyonrails.org/classes/ActiveSupport/Testing/ConstantStubbing.html#method-i-stub_const